resources/views/admin/partials/header.blade.php

```diff
@@ -9,15 +9,10 @@
         <li class="nav-item dropdown">
             <a role="button" class="dropdown-toggle nav-link">
                 <span>
-                    {{-- TODO support image as avatar --}}
-                    {{--<img src="" alt="admin@bootstrapmaster.com" class="img-avatar">--}}
-
-                    @if(Auth::check() && Auth::user()->first_name && Auth::user()->last_name)
-                        <span class="avatar-initials">{{ mb_substr(Auth::user()->first_name, 0, 1) }}{{ mb_substr(Auth::user()->last_name, 0, 1) }}</span>
-                    @elseif(Auth::check() && Auth::user()->name)
-                        <span class="avatar-initials">{{ mb_substr(Auth::user()->name, 0, 1) }}</span>
-                    @elseif(Auth::guard(config('admin-auth.defaults.guard'))->check() && Auth::guard(config('admin-auth.defaults.guard'))->user()->first_name && Auth::guard(config('admin-auth.defaults.guard'))->user()->last_name)
-                        <span class="avatar-initials">{{ mb_substr(Auth::guard(config('admin-auth.defaults.guard'))->user()->first_name, 0, 1) }}{{ mb_substr(Auth::guard(config('admin-auth.defaults.guard'))->user()->last_name, 0, 1) }}</span>
+                    @if(Auth::check() && Auth::user()->avatar_url)
```

Comment:
This part is definitely no ok. You need also the old checks, because of backward compatibility with User model for auth, which will not have the abbr. Also I recommend to use whole words as properties and methods and no just shortcuts.